redaxo/src/core/lib/login/login.php

```diff
@@ -495,19 +497,27 @@ private static function rewriteSessionCookie($sameSite)
 
     /**
      * Verschlüsselt den übergebnen String.
+     *
+     * @return null|string
```

Comment:
```suggestion
     * @return string
```

Wieder so ein Fall, wo ich nicht weiß, ob man den Typ von PHP 1:1 weitergeben sollte, oder nicht eher den Typ deklarieren sollte, den man eigentlich sicher haben möchte.

Reply:
Wenn wir string haben wollen müssen wir das evtl entstehende null abfragen und mittels exception habdeln